project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/TopProductDataFixture.php

```diff
@@ -17,39 +17,58 @@ class TopProductDataFixture extends AbstractReferenceFixture implements Dependen
      */
     protected $topProductFacade;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
     /**
      * @param \Shopsys\FrameworkBundle\Model\Product\TopProduct\TopProductFacade $topProductFacade
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      */
-    public function __construct(TopProductFacade $topProductFacade)
+    public function __construct(TopProductFacade $topProductFacade, Domain $domain)
     {
         $this->topProductFacade = $topProductFacade;
+        $this->domain = $domain;
     }
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
      */
     public function load(ObjectManager $manager)
     {
-        $topProductReferenceNames = [
+        $topProductReferenceNamesOnAllDomainsExcludingSecond = [
```

Comment:
I don't know if I like the name of this.. It could be used on second domain as well but its default for every other one.. maybe something like `defaultTopProduct...`?

Reply:
fixed in dc60b55e4